videoio.go

```diff
@@ -204,8 +205,11 @@ func (v *VideoCapture) IsOpened() bool {
 
 // Read reads the next frame from the VideoCapture to the Mat passed in
 // as the param. It returns false if the VideoCapture cannot read frame.
-func (v *VideoCapture) Read(m *Mat) bool {
-	return C.VideoCapture_Read(v.p, m.p) != 0
+func (v *VideoCapture) Read(m *Mat) (err error) {
+	if C.VideoCapture_Read(v.p, m.p) == 0 {
+		err = errors.New("VideoCapture Read error")
```

Comment:
Oh, the library doesn't return an enum for error description to be able to describe the error at least a bit? :/

Reply:
OpenCV returns a bool here, so not much we can do about it.

Reply:
Oh I see, this is kinda disappointing. I wonder if there is much value into the PR then for the code churn it represents. I naively thought there would be some form of error code. :/

Reply:
Agreed. I will probably just close it then to avoid the hassle to everyone.